scripts/docGenerator.py

```diff
@@ -0,0 +1,160 @@
+import os
+import subprocess
+import shutil
+
+# Read the PSP_PATH environment variable
+try:
+    PSP_PATH = os.environ['PSP_PATH']
+    print(f"Using '{PSP_PATH}' as the PSP_PATH")
+except:
+    print("Please set the PSP_PATH environment variable.\nExiting.")
+    exit()
+
+# Commands to use
+GIT_ADD_CMD = 'git add .'
+GIT_COMMIT_CMD = 'git commit -m "Updating documentation automatically from docGenerator.py"'
```

Comment:
I'm not sure if I like this commit message or not. What are your thoughts @seanboltjes ?

Reply:
We'll keep it for now can adjust later